modules/doc/content/newsletter/2021_01.md

```diff
@@ -0,0 +1,22 @@
+# MOOSE Newsletter (January 2021)
+
+# Improved INL HPC Remote Connectivity
+
+The [hpc_remote.md] instructions have been updated for a better user experience in accessing web-based INL HPC resources. The previous instructions involved using an additional browser with a proxy to access said resources (hpcgitlab, moosebuild, hpcweb, etc). The new instructions utilize a proxy configuration that is much easier to setup for the end-user and do not require an additional browser. For more information, see [hpc_remote.md#socks-proxy].
+
+# Ray Tracing Module
+
+!media large_media/ray_tracing/cone_ray_study_rays.png style=width:32%;float:right;margin-left:10px;;margin-right:10px
+
+Native ray tracing capability has been released in the [ray_tracing/index.md].
+
+The module traces rays throughout the finite element mesh. Notable features include:
+
+- Ray contribution to residuals and Jacobians along ray segments (example: line sources)
+- Tracing on meshes with adaptivity
+- Supports ray interaction with internal and external boundaries
+- Supports storage and manipulation of data unique to each ray
+- Supports ray interaction with field variables
+- Supports parallel execution
+
+The pictured example is an anisotropic point source that emits in a cone of directions. See [flashlight_source.md] for more information.
```

Comment:
"Cone of directions" sounds a bit awkward to me, without considering the pictured example. Which directions, or what direction?

Reply:
```
The pictured example is an anisotropic point source that is nonzero only in a cone
```

How about that?

Reply:
I think that sounds better. The fact that this cone is made up of multiple rays with multiple directions is explained in the linked doco, so keeping this a tiny bit more general is probably best for a newsletter.

Reply:
I opted to just remove it. Pretty picture is enough as it is detailed on the main module page.